### PR TITLE
Support for Windows and macOS

### PR DESCRIPTION
**Pull request by db7**
This PR fixes the file paths on windows and adds rules to create windows and darwin binaries on releases.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,8 @@ We assume that your user has rights to run docker, ie, either with rootless
 Docker or having your user in the `docker` group.  Please read the Docker
 [postinstall instructions][] further instructions.
 
-> Note: we have not yet tested `vsyncer` in macOS or Windows.
+> Note: we have `vsyncer` binaries for macOS or Windows, but they haven't
```
Comment (hernanponcedeleon) [inline README.md:35]: "macOS or Windows" ->  macOS and Windows

File: README.md [hunk 2]
```diff
@@ -47,14 +48,14 @@ releases use the fixed tag of the release.
...
-    make install PREFIX=/path/to/bin DOCKER_TAG=v2.0.0
+    make install PREFIX=/path/to/bin DOCKER_TAG=latest
 
 ### Disabling Docker
 
 To run `vsyncer` without Docker, you'll need the following tools:
 
 - clang and llvm >= v14
-- Dartagnan >= v4.0.0 (alternative)
+- Dartagnan >= v4.0.1 (alternative)
```
Comment (hernanponcedeleon) [inline README.md:58]: I would prefer not to have this until we actually make the release. I want to merge just one more PR from Thomas, ask you to test it in the demo and then I will release.

File: tools/docker.go [hunk 3]
```diff
@@ -41,46 +41,34 @@ func DockerPull(ctx context.Context) error {
...
-			return fmt.Errorf("could get user groups: %v", err)
-		} else if !strings.Contains(string(output), "docker") {
-			return fmt.Errorf("user is not in docker group")
+	// find out current directory
+	if cwdRel, err := os.Getwd(); err != nil {
+		return err
+	} else if cwdAbs, err := filepath.Abs(cwdRel); err != nil {
+		return fmt.Errorf("could not find volume path '%s': %v", cwdRel, err)
+	} else {
+		// fix windows paths to contain only slashes
```
Comment (hernanponcedeleon) [inline tools/docker.go:66]: "fix" means we need to fix, or the code below "fixes windows"?

File: tools/docker.go [hunk 4]
```diff
@@ -94,6 +82,14 @@ func DockerRun(ctx context.Context, args []string, volumes []string) error {
 		if abs, err := filepath.Abs(v); err != nil {
 			return fmt.Errorf("could not find volume path '%s': %v", v, err)
 		} else {
+			// fix windows paths to contain only slashes
```
Comment (hernanponcedeleon) [inline tools/docker.go:85]: Same as above. Can you extract a method for this to avoid repetition?

Comment (db7) [reply tools/docker.go:82]: Done

**Review by hernanponcedeleon — APPROVED** [review]

**Pull request merged**